File: src/main/java/refactown/cleancode/c03functions/CommandQuery.java
package refactown.cleancode.c03functions;

public class CommandQuery {

	public static boolean sett(String attribute, String value) {
		return true;
	}

	public static boolean attributeExists(String attribute) {
		return true;
	}
	
	public static void setAttribute(String attribute, String value) {
		
	}

	public static void main(String[] args) {
		if (sett("username", "bill")) {}

	    if (attributeExists("username")){
	        setAttribute("username", "bill");
	    }
	}
}

File: src/main/kotlin/refactown/cleancode/c03functions/CommandQuery.kt
package refactown.cleancode.c03functions


fun sett(attribute: String, value: String): Boolean    = true

fun attributeExists(attribute: String): Boolean        = true

fun setAttribute(attribute: String, value: String)     = print("")

fun main() {
    if (sett("username", "bill")) {}

    if (attributeExists("username")){
        setAttribute("username", "bill")
    }
}